package.json

```diff
@@ -30,6 +30,7 @@
     "clean": "lerna clean --yes && rm -rf node_modules",
     "clean:install": "npm run clean && npm install",
     "jest": "jest",
+    "wdio": "node ./packages/terra-functional-testing/bin/test-runner.js",
```

Comment:
I wasn't sure how to hoist the `tt-test` command from terra-functional-testing to the root of the lerna repository. Its the reverse of the exposing the parent bin commands to the packages. If anyone knows how to get this working let me know and I'll update it.

Reply:
This is what I had to do as well for some of the stuff I've been playing around with.  No problems on my end with it.